### ISSUE TITLE
Migrate golang.org/x/crypto to standard library imports

### ISSUE THREAD

**Issue by HACKERALERT** [HACKERALERT/Picocrypt#177]
https://github.com/golang/go/issues/65269

This means that all cryptography will be soon be done by the standard library. Hooray!

**Comment by HACKERALERT** [comment]
@orochi02 The standard library has the best support, security, and reliability, so once golang.org/x/crypto/* is merged into crypto/*, now the cryptography that Picocrypt uses benefits from the standard library "treatment", will continue to get timely updates, and users building from source get the latest patches without me having to update go.mod. Overall, there's no reason not to do this.

**Comment by HACKERALERT** [comment]
Moved to Changelog's Future list
